### PR TITLE
Zj1.6 about

### PR DESCRIPTION
**Pull request by michaljedro**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -74,6 +74,26 @@ <h1>Lorem ipsum dolor sit amet.</h1>
...
     </div>
   </div>
+  <div class="bgc_1">
+    <div class="about wrap">
+      <div class="about__img">
+
+      </div>
+      <div class="about__text">
+        <p class="about__text-paragraph">Lorem ipsum dolor sit amet.</p>
+        <p class="">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptates aut adipisci officia fugit
```
Comment (pawel-schmidt) [inline index.html:84]: Remove empty `class` attribute.

**Review by pawel-schmidt — CHANGES_REQUESTED** [review]
Please attach some screenshot of your work :-)

File: app/scss/_global.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.wrap {
+    border: 1px solid orange;
+    min-width: 80%;
+    max-width: 1200px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.bgc_1 {
```
Comment (pawel-schmidt) [inline app/scss/_global.scss:9]: Names with a number suffix are a bad practice. There are of course some exceptions, but this one isn't one ;-)

Comment (pawel-schmidt) [reply app/scss/_global.scss:9]: Actually, is it used anywhere?

**Review by pawel-schmidt — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (michaljedro) [reply index.html:84]: Removed ;)

File: app/scss/_global.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.wrap {
+    border: 1px solid orange;
```
[the remark this answers is not in the document]
Comment (michaljedro) [reply app/scss/_global.scss:2]: Pardon

File: app/scss/_global.scss (hunk 2 above)
Comment (michaljedro) [reply app/scss/_global.scss:9]: i used once. I decided to add this class after our conversation in Warsaw. I supose to used as a class with background color.

Comment (michaljedro) [reply app/scss/_global.scss:9]: i think, the best way would be to remove it and use scss $ sign.

Comment (michaljedro) [reply app/scss/_global.scss:8]: ok. i think that class bgc_1 is not necessary.

**Review by pawel-schmidt — APPROVED** [review]

**Pull request merged**